Components/FlightControl/RocketSM.cpp

```diff
@@ -483,7 +483,7 @@ RocketState Ignition::OnEnter()
     // Assert vent and drain closed, and MEV enable pin on
```

Comment:
edit this comment. the mev enable pin will be off

Reply:
all the enable comments have been changed